project-metrics/metrics_service/database/models_test.py

```diff
@@ -0,0 +1,68 @@
+"""Tests for database models."""
+
+from absl import flags
+from absl.testing import flagsaver
+import datetime
+import unittest
+from unittest import mock
+import sqlalchemy
+from sqlalchemy import orm
+import sys
+
+from database import init_db
+from database import models
+
+FLAGS = flags.FLAGS
+FLAGS(sys.argv)
+
+
+class TestBuild(unittest.TestCase):
+
+  @classmethod
+  def setUpClass(cls):
+    super(TestBuild, cls).setUpClass()
+
+    engine = sqlalchemy.create_engine('sqlite:///:memory:', echo=True)
+    init_db.init_db(engine)
+    cls.Session = orm.scoped_session(orm.sessionmaker(bind=engine))
+
+    cls.FAKE_NOW = datetime.datetime(2019, 6, 1)
+    cls.FAKE_90_DAYS_AGO = cls.FAKE_NOW - datetime.timedelta(days=90)
```

Comment:
I'm wondering if you really need to fix the dates in place, since they are relative

If you don't mock the `datetime.datetime.now` method and just create the db rows with the real "now" and "now - 90 days", does this fail?

Reply:
Done